linkerd/app/src/env.rs

```diff
@@ -169,10 +169,6 @@ const ENV_DNS_MIN_TTL: &str = "LINKERD2_PROXY_DNS_MIN_TTL";
 /// Lookups with TTLs above this value will use this value instead.
 const ENV_DNS_MAX_TTL: &str = "LINKERD2_PROXY_DNS_MAX_TTL";
 
-/// The amount of time to wait for a DNS query to succeed before falling back to
-/// an uncanonicalized address.
-const ENV_DNS_CANONICALIZE_TIMEOUT: &str = "LINKERD2_PROXY_DNS_CANONICALIZE_TIMEOUT";
```

Comment:
does this mean that refinements will no longer time out?

Reply:
This snuck in from a followup and should have been reverted